packages/shared/providers.yaml

```diff
@@ -466,6 +466,30 @@ contentstack:
     authorization_url: https://${connectionConfig.subdomain}.contentstack.com/apps/${connectionConfig.appId}/authorize
     token_url: https://${connectionConfig.subdomain}.contentstack.com/apps-api/apps/token
     docs: https://docs.nango.dev/integrations/all/contentstack
+#Untested configuration. Please reach out if you have a test account that we can use to test it.
+datev:
+    categories:
+        - legal
+        - payroll
+        - hr
+    auth_mode: OAUTH2
+    authorization_url: https://login.datev.de/openid/authorize
+    token_url: https://api.datev.de/token
+    token_request_auth_method: basic
+    scope_seprator: ' '
```

Comment:
Just asking, but doesn't that typo have any side effects?

Reply:
good catch, I'll fix that 👌🏻

Reply:
There's another one in another section btw. Thanks